### ISSUE TITLE
detect small objects

### ISSUE THREAD

**Issue by tbnxyzfz1** [matterport/Mask_RCNN#2707]
I want to detect two classes of objects in my dataset. The size of the images is 320x256. Class 1 objects are of normal size. The size of most class 2 objects is 2x2 pixels. I wonder which parameter can I change to improve the identification result of class 2 objects? I changed RPN_ANCHOR_SCALES to (2, 4, 8, 64, 128). The loss of the trained model is less than 0.7. But the trained model still can't detect class 2 objects in the test dataset. Thank you in advance.

**Comment by nataliameira** [comment]
@tbnxyzfz1 
Do you have a lot of small objects in the scene?

**Comment by tbnxyzfz1** [comment]
> @tbnxyzfz1 Do you have a lot of small objects in the scene?

No, only <10 small objects in one image. Sometimes no small objects in the images.
